modules/agent/pkg/controllers/bundledeployment/controller.go

```diff
@@ -252,3 +265,24 @@ func readyError(status fleet.BundleDeploymentStatus) error {
 
 	return errors.New(msg)
 }
+
+func hasAutoReapply(bd *fleet.BundleDeployment, status *fleet.BundleDeploymentStatus) (ok bool, err error) {
+	if bd.Spec.Options.AlwaysReapply || bd.Spec.StagedOptions.AlwaysReapply {
+		if status.ReapplyAfter == "" {
+			status.ReapplyAfter = time.Now().Add(5 * time.Minute).Format(time.RFC3339)
+			return ok, nil
+		}
+
+		reapplyAfter, err := time.Parse(time.RFC3339, status.ReapplyAfter)
+		if err != nil {
+			return ok, err
+		}
+		if reapplyAfter.Before(time.Now()) {
+			status.ReapplyAfter = ""
+			ok = true
+		}
+
+	}
+
+	return ok, nil
```

Comment:
I understand your code here, but for readability, could we use literal values instead? `false` over unset `ok`?